pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -34,6 +34,24 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	// The constant is used if sync_nodes_status=False. NodeController will not proactively
+	// sync node status in this case, but will monitor node status updated from kubelet. If
+	// it doesn't receive update for this amount of time, it will start posting node NotReady
+	// condition. The amount of time when NodeController start evicting pods is controlled
+	// via flag 'pod_eviction_timeout'. Note: be cautious when changing nodeMonitorGracePeriod,
```

Comment:
I've tried to not have the side effect: pod_eviction_timeout is user facing, we shouldn't add an extra time on that (here, monitorGracePeriod).  So instead of counting from node becomes NotReady, we count the age of last node Ready.

Yes, pod_eviction_timeou' starts counting from when the node becomes not ready.  But I've tried not to make pod_eviction_time added upon